page-supporters.php

```diff
@@ -77,6 +77,10 @@
                                      'id'=>'16','linkedin'=>'https://www.linkedin.com/company/learnupon',
                                              'twitter'=>'https://twitter.com/learnupon'],
 
+                             ['src'=>'../wp-content/themes/cd-theme/static/img/supporters/pivotal.png', 'name'=>'Pivotal', 'jobDescription'=>'The Pivotal Dublin team provided pro-bono project managers, designers and developers to work on a project relating to the registration and events flow for our Community Platform.',
```

Comment:
More of a question for @rosalanghammer.

Should this Pivotal, or specifically call out Pivotal Labs?